README.md

```diff
@@ -65,12 +70,16 @@ bm buildbot notify your@email.address
 
 iOS and Android applications allow for evergreen updates. This option will allow for resouces to be zipped into a folder per platform 
 
+The `evergreen` command will also upload your zip files to a remote location to allow your applications to download them.
+Uploading your zip files requires users to be authenicated with a BlinkMobile Identity.
+To login to the Buildbot CLI, see [identity-cli](https://github.com/blinkmobile/identity-cli)
+for available commands.
+
 `bm buildbot evergreen` supports the following options:
-- `--force` (optional, overwrite any existing update ZIPs)
+- `--force` optional, overwrite any existing update zip files
+- `--no-upload` optional, prevent the evegreen zip files from being uploaded to remote location
 
-To zip resources for a project:  
+To zip resources for a project and upload:  
 ```sh
-bm buildbot evergreen <path-to-project-root> [--force]
+bm buildbot evergreen <path-to-project-root> [--force] [--upload]
```

Comment:
it confused me because the upload behavior is automatic but then we are saying 'if you want to upload it use the `--upload` flag', and that is the only place in the docs the flag is mentioned. Perhaps leave this as is and documenting the upload flag?